### DISCUSSION TITLE
Download is not function again

### DISCUSSION THREAD

**Discussion by Papermint155** [pear-devs/pear-desktop#2721] [Q&A] [answered]
When I clicked download the message kept jumping out with the 2xx error.
I used it 2 weeks ago when the download function worked very well at that time.
So, is there any solution for this error or just server problems?
![Screenshot 2024-12-16 183630](https://github.com/user-attachments/assets/893a4542-ae60-455c-a871-b81a52772c20)

**Comment by spoonh** [comment]
Hey mate assuming youre on windows 11 i actually built the latest commit from a few hours ago into an exe, lmk if u want!

**Reply by spoonh** [reply, marked as answer]
https://github.com/th-ch/youtube-music/discussions/2726

**Reply by Papermint155** [reply]
thank you. Really appreciate your help.

**Reply by spoonh** [reply]
of course! lmk in the other thread if u need another distro built i have all the stuff still setup in my environment

**Comment by JellyBrick** [comment]
Duplicate of #2691, #2628, #2700
